src/Language/Haskell/HGrep/Prelude.hs

```diff
@@ -109,6 +111,10 @@ module Language.Haskell.HGrep.Prelude (
   , on
   , seq
 
+  -- * Predicates
+  -- ** Char
+  , isSpace
```

Comment:
Sorry to be fussy, could we keep `all` but scrap `isSpace` from here?

I'll write up a couple of guidelines for Prelude inclusion, but in general I try to keep only very general or very commonly used functions in the project Prelude. For example, for things like List and Char, the qualified `List.filter` and `Char.isSpace` forms make code much easier to follow.

Reply:
Not a worry, totally makes sense! I'll make the change